sympy/integrals/laplace.py

```diff
@@ -1089,6 +1092,211 @@ def simp_exp(arg):
     return _simplify(f.subs(t, t_), simplify), cond
 
 
+def _complete_the_square_in_denom(f, s):
+    from sympy.simplify.radsimp import fraction
+    [n, d] = fraction(f)
+    if d.is_polynomial(s):
+        cf = d.as_poly(s).all_coeffs()
+        if len(cf)==3:
+            a, b, c = cf
+            d = a*((s+b/(2*a))**2+c/a-(b/(2*a))**2)
+    return n/d
+
+
+def _inverse_laplace_build_rules(s, t):
+    """
+    This is an internal helper function that returns the table of inverse
+    Laplace transform rules in terms of the time variable `t` and the
+    frequency variable `s`.  It is used by `_inverse_laplace_apply_rules`.
+    """
+    a = Wild('a', exclude=[s])
+    b = Wild('b', exclude=[s])
+    c = Wild('c', exclude=[s])
+    d = Wild('d', exclude=[s])
+    global _ILT_vars
+    global _ILT_rules
+
+    if _ILT_vars == (s, t):
+        return
+
+    debug('_inverse_laplace_build_rules with (%s, %s)'%(s, t))
```

Comment:
I'm not sure if the debug are evaluated lazy, 
but it is likely that it can eagerly evaluate strings even if `SYMPY_DEBUG` is not set to True.

Reply:
It does not matter whether it is lazy because '_inverse_laplace_build_rules with (%s, %s)'%(s, t) is evaluated right away. This is a weakness of this type of debugging, I will take it into the task list. I would leave it for now since it does not cost a lot of time and has been in the code for quite some time now.

Reply:
This is now in the work list; I resolve the conversation